.github/workflows/tests.yml

```diff
@@ -54,3 +54,24 @@ jobs:
               with:
                 flags: php-${{ matrix.php-version }}-${{ matrix.os }}
                 name: php-${{ matrix.php-version }}-${{ matrix.os }}
+
+    analyse-php:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - name: Use php 7.3
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: 7.3
+                  tools: composer:v2
+            - name: Cache module
+              uses: actions/cache@v2
+              with:
+                  path: ~/.composer/cache/
+                  key: composer-cache
+            - name: Install phpstan
+              run: composer require --dev phpstan/phpstan
+            - name: Install doctum
```

Comment:
Installing doctum makes no sense

Reply:
for phpstan to analyse scripts/ as well ;-)